ci/baremetal/baremetal-cluster.tf.envsubst

```diff
@@ -0,0 +1,42 @@
+module "baremetal" {
+  source = "../../bare-metal/flatcar-linux/kubernetes"
+
+  providers = {
+    local = "local.default"
+    null = "null.default"
+    template = "template.default"
+    tls = "tls.default"
+  }
+
+  cached_install = "true"
+  matchbox_http_endpoint = "http://matchbox.example.com:8080"
+
+  cluster_name = "mercury"
+  k8s_domain_name = "node1.example.com"
+
+  ssh_authorized_key = "$PUB_KEY"
```

Comment:
It has to be changed after https://github.com/kinvolk/lokomotive-kubernetes/pull/98.

Reply:
Thanks